components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -381,6 +397,19 @@ void Nudging::run_impl (const double dt)
   // Perform horizontal remap (if needed)
   m_horiz_remapper->remap(true);
 
+  // bypass copy_and_pad and vert_interp for hybrid coordinate:
```

Comment:
I'm confused. I thought the hybrid approach was based on reconstructing p_mid_src from ps/hyam/hybm (after horiz interp), and then proceed to do vert interp. That's also what SPA does. I don't see ps/hyam/hybm used here, while in the hpp I saw the comment
```cpp
  // hybrid data includes hyai(ilev),hybi(ilev),hyam(lev),hybm(lev),ilev(ilev),lev(lev),P0,PS(time, ncol)
```
To me, it seems like this hybrid approach is simply an horiz interpolation, assuming the data levels are _the same_ as the model levels.

This, btw, brings back a discussion that @AaronDonahue started: horiz interpolation should happen on pressure levels, not model levels.

Reply:
or should i change the name for this condition? HYBRID does make people think it's need reconstructing p_mid_src from the hybrid coordinate...

Reply:
What if, instead of specifying a new value for the src pres type, we use a _separate_ input option. After all, the src pressure is still a time dependent 3d profile. What about adding the following?
```
  <skip_vert_interpolation type="logical" doc="blahblah">false</skip_vertical_interpolation>
```
Clearly, if this is set to `true`, we need to check that `m_num_levs==m_src_num_levs`, otherwise we _can't_ skip vert interpolation. We could also check that this option is not set to `true` if the static 1d profile is used (makes no sense to not interpolate if src press is uniform and constant).

Reply:
thanks for the suggestion! it makes sense to me. I will change them